projects/demo-playwright/tests/kit/action-bar/action-bar.spec.ts

```diff
@@ -17,4 +17,15 @@ test.describe('ActionBar', () => {
 
         await expect(actionBarExample).toHaveScreenshot('01-actions-bar.png');
     });
+
+    test('should show on top', async ({page}) => {
+        await tuiGoto(page, DemoRoute.ActionBar);
+        const example = page.locator('#top-position');
+        const showActionBarButton = example.getByTestId('open-table-bar-on-top').first();
+
+        await showActionBarButton.click();
```

Comment:
The reason for the errors below is that I am adding a test to check if the position is on top